docs/get-started.md

```diff
@@ -91,6 +90,21 @@ export OTEL_EXPORTER_OTLP_ENDPOINT=https://my-deployment.apm.us-west1.gcp.cloud.
 export OTEL_EXPORTER_OTLP_HEADERS="Authorization=Bearer P....l"
 ```
 
+> [!NOTE]
+> Alternatively, you can use an [APM agent key](https://www.elastic.co/guide/en/observability/current/apm-api-key.html) to authorize requests to an Elastic Observability endpoint. APM agent keys are revocable, you can have more than one of them, and you can add or remove them without restarting APM Server.
+>
+> To create and manage APM Agent keys in Kibana:
+>
+> 1. Go to **APM Settings**.
+> 1. Select the **Agent Keys** tab.
+>
+> When using an APM Agent key, the `OTEL_EXPORTER_OTLP_HEADERS` is set using different auth schema (`ApiKey` rather than `Bearer`). For example:
+> ```sh
+> export OTEL_EXPORTER_OTLP_ENDPOINT=https://my-deployment.apm.us-west1.gcp.cloud.es.io
```

Comment:
Since this should not be different we can drop this line? Or we want to have a full example we should add
`export OTEL_RESOURCE_ATTRIBUTES=service.name=<app-name>`